mailpoet/assets/js/src/subscribers/stats/no-access-info.tsx

```diff
@@ -42,10 +46,25 @@ export function NoAccessInfo(): JSX.Element {
     >
       <thead>
         <tr>
-          <th>{MailPoet.I18n.t('email')}</th>
-          <th>{MailPoet.I18n.t('columnAction')}</th>
-          <th>{MailPoet.I18n.t('columnCount')}</th>
-          <th>{MailPoet.I18n.t('columnActionOn')}</th>
```

Comment:
When removing placeholder-based translations, could you please remove them from the views too if they're not used anywhere else?

Reply:
Good catch, I didn't think of that.
I was able to remove 2, the others were still in use in other places.